RevenueCatUI/Modifiers/ViewExtensions.swift

```diff
@@ -90,8 +90,6 @@ extension View {
         if enabled {
             if #available(iOS 16.0, macOS 13.0, tvOS 16.0, watchOS 9.0, *) {
                 ViewThatFits(in: axis.scrollViewAxis) {
-                    self
-
```

Comment:
This... makes me nervous to remove even thought it does look weird that `self` is in here twice 😬 

I feel like it had to provide some value so I'm going to go back into the PRs to see if there is any knowledge there